### DISCUSSION TITLE
Customization for User list in Administrator menu

### DISCUSSION THREAD

**Discussion by queza85** [exoplatform/exo-community#60] [Q&A]
Hi 

I have a question that I customize user list in administrator menu.
I would like add some fields and increase user count per page in user management list.

But when I check the sample document and extension source, I cannot find any related about this.
Do you provide any extension for this?

**Comment by rdenarie** [comment]
Hello @queza85 
No we have not extension point on this 2 items

**Reply by queza85** [reply]
I see. is it possible to create new portlet and add this to page as portlet?

maybe that portlet should call user api directly. but my concern is how to use auth for that api automatically.

**Reply by rdenarie** [reply]
> I see. is it possible to create new portlet and add this to page as portlet?

Yes it is

> how to use auth for that api automatically.

When call API endpoint, use javascript fetch, with including credentials. So that, Rest endpoint are called with the current user context. Example  : https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/js/UserService.js#L68-L79

**Reply by queza85** [reply]
thank you for your answer.

I will try to develop as a new portlet to manage user list.

By the way, does it have any repo for basic user list portlet in administrator?

**Reply by rdenarie** [reply]
Hello @queza85 
I'm sorry, I don't understand your last question.

**Reply by queza85** [reply]
Hi, 

I mean EXO provides this portlet source for user management?

If it is provided, I can start to change only a few things here.

<img width="2596" alt="image" src="https://github.com/exoplatform/exo-community/assets/2861131/da998588-8ff8-405b-871c-ccac68da2e9b">

**Reply by rdenarie** [reply]
The user Management component is available here : 
https://github.com/Meeds-io/social/tree/develop/webapp/portlet/src/main/webapp/vue-apps/idm-users-management

**Reply by queza85** [reply]
One more question that it has any cache?
when I change script side, but it looks didn't change. 
but UI was changed at once. I just copy and overwrite war and then restart server only.

I think above issue related with this. when I add console and alert script but it didn't deployed except UI side.

**Reply by queza85** [reply]
I found that is related with browser cache. so I use incognito mode
